nextflow.config

```diff
@@ -110,7 +110,7 @@ profiles {
         podman.enabled         = false
         shifter.enabled        = false
         charliecloud.enabled   = false
-        channels               = ['conda-forge', 'bioconda', 'defaults']
```

Comment:
What did this needed to be changed? It seemed at other locations it's still used as before?

Reply:
This seems to be a silent bug since dec 2022! I already posted this on slack: https://nfcore.slack.com/archives/CE5LG7WMB/p1714721754644179

Without the `conda.` it is not considered for the creation of the environment! This is probably also an issue in nf-core/epitopeprediction! @jonasscheid